reddit_service_websockets/dispatcher.py

```diff
@@ -22,6 +46,14 @@ def __init__(self, metrics):
     def on_message_received(self, namespace, message):
         consumers = self.consumers.get(namespace, [])
 
+        # Compress the message
+        compressed = make_compressed_frame(message, COMPRESSOR)
+        message = Message(
+            compressed=compressed,
+            raw=message,
+            percent_compressed=1 - float(len(compressed))/float(len(message)),
```

Comment:
Hmm, yeah, I guess for live comments / live threads it wont be trivial.  What about a sampling rate?  Or: I'm not sure how we could aggregate and not have smaller namespaces step over the larger ones (for instance, a single orangered namespace might aggregate to a savings of 10%, where a 70k comment thread namespace might aggregate to 50%.  The 50% should completely dwarf the 10% one, but wouldn't if we just sent one metric back per message per namespace).